plugins/synology-ss/src/main.ts

```diff
@@ -0,0 +1,283 @@
+import sdk, { ScryptedDeviceBase, DeviceProvider, Settings, Setting, ScryptedDeviceType, VideoCamera, MediaObject, Device, ScryptedInterface, Camera, MediaStreamOptions, PictureOptions } from "@scrypted/sdk";
+import { recommendRebroadcast } from "../../rtsp/src/recommend";
+import {SynologyApiClient, SynologyCameraStream, SynologyCamera} from "./api/synology-api-client";
+
+const { log, deviceManager, mediaManager } = sdk;
+
+class SynologyCameraDevice extends ScryptedDeviceBase implements Camera, Settings, VideoCamera {
+    private provider: SynologySurveillanceStation;
+    private streams: SynologyCameraStream[];
+
+    constructor(provider: SynologySurveillanceStation, nativeId: string, camera: SynologyCamera) {
+        super(nativeId);
+        this.provider = provider;
+
+        this.streams = [
+            { ...camera.stream1, id: '1' },
+            { ...camera.stream2, id: '2' },
+            { ...camera.stream3, id: '3' },
+        ].filter(s => !!s.resolution);
+
+        this.motionDetected = false;
+        if (this.interfaces.includes(ScryptedInterface.BinarySensor)) {
```

Comment:
might wanna just remove this check, since I don't think Synology has doorbell devices (binary sensor is the doorbell button)